suite-native/device-manager/src/components/DeviceItem/DeviceItemContent.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { HStack, Box } from '@suite-native/atoms';
+import { Translation, useTranslate } from '@suite-native/intl';
+import {
+    selectDeviceByState,
+    DeviceRootState,
+    PORTFOLIO_TRACKER_DEVICE_ID,
+} from '@suite-common/wallet-core';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { DeviceItemIcon } from './DeviceItemIcon';
+import { SimpleDeviceItemContent } from './SimpleDeviceItemContent';
+import { WalletDetailDeviceItemContent } from './WalletDetailDeviceItemContent';
+import { DeviceItemContentProps } from '../../types';
```

Comment:
Please move the props to the component. It should be defined in the file where it's used...